config/processors/syslog_log_audit_checkpoint.fw.conf

```diff
@@ -11,6 +11,10 @@ filter {
     # dropping timestamp from collector logstash so that it's regenrated in indexer
     remove_field => ["log_source_type", "@timestamp"]
   }
+# Parsing the syslog pri header if present
+  grok {
+    match => { "message" => "^.*?<(?<pri>\d+)>(\s)?(?<rest_msg>.*?)%" }
+  }
 
```

Comment:
add syslog_pri part

Reply:
added syslog_pri field in .conf files, please check.